common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -99,6 +65,62 @@ def instantiate_descriptor(**field_data):
     )
 
 
+def set_up():
```

Comment:
setUp and tearDown are methods that one overrides from classes that inherit from unitest.TestCase (see https://docs.python.org/2/library/unittest.html). This is exactly the sort of thing that they are designed to perform for you.